### DISCUSSION TITLE
Possible to auto-start all containers on dockge first-run

### DISCUSSION THREAD

**Discussion by EJSRobinson** [louislam/dockge#296] [Feature Request]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I am deploying dockge via a build-script onto cloud-based machines. 
As such, I trying to make it such that when dockge starts for the first time (or any other time), it will also start the child containers. Note that these child containers are also pre-defined in my build-script and therefore within in dockge.

This is so every time I create a new server, I do not have to go through the process of browsing to dockge, making an account, and starting the child containers.

Thanks 🙂

### 🐻 Dockge Version

1.2.0

### 💻 Operating System and Arch

ubuntu 22.04 x64

### 🌐 Browser

Google Chrome

### 🟩 NodeJS Version

20.x

**Comment by louislam** [comment]
I see your point. I change it to feature-request.


> NodeJS Version
> 20.x

Just a friendly remind, due to the bug of the terminal library, it supports up to Node.js 18.17.1 only.

You will encounter cpu 100% usage on 20.x.

Read more: #186

**Reply by gbrandt1** [reply]
Hello,

Instead of new feature request, i want to expand this one:

How about adding a "All" toggle switch to make operations on the top buttons work on all stacks at the same time?
